configurations/manager/backup-1TB-gce.yaml

```diff
@@ -0,0 +1,40 @@
+             "cassandra-stress write         cl=QUORUM duration=620m -schema 'replication(factor=3) compression=SnappyCompressor  compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=snappy                -rate threads=10 -pop seq=1..50000000    -log interval=5",
+             "cassandra-stress write         cl=QUORUM duration=630m -schema 'replication(factor=3) compression=DeflateCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none                  -rate threads=10 -pop seq=1..50000000    -log interval=5",
+             "cassandra-stress write         cl=QUORUM duration=630m -schema 'replication(factor=3) compression=ZstdCompressor    compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none                  -rate threads=10 -pop seq=1..50000000    -log interval=5",
+             "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=690m                                                  -port jmx=6868 -mode cql3 native                                   -rate threads=10"]
+
+stress_read_cmd: ["cassandra-stress read cl=QUORUM duration=690m                                                                                                                 -port jmx=6868 -mode cql3 native  -rate threads=10 -pop seq=1..1100200300  -log interval=5 -col 'size=FIXED(200) n=FIXED(5)'",
+                  "cassandra-stress read cl=QUORUM duration=600m -schema 'replication(factor=3) compression=LZ4Compressor     compaction(strategy=SizeTieredCompactionStrategy)'     -port jmx=6868 -mode cql3 native compression=lz4                   -rate threads=20 -pop seq=1..50000000    -log interval=5",
+                  "cassandra-stress read cl=QUORUM duration=620m -schema 'replication(factor=3) compression=SnappyCompressor  compaction(strategy=SizeTieredCompactionStrategy)'  -port jmx=6868 -mode cql3 native compression=snappy                -rate threads=20 -pop seq=1..50000000    -log interval=5",
+                  "cassandra-stress read cl=QUORUM duration=630m -schema 'replication(factor=3) compression=DeflateCompressor compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none                  -rate threads=20 -pop seq=1..50000000    -log interval=5",
+                  "cassandra-stress read cl=QUORUM duration=630m -schema 'replication(factor=3) compression=ZstdCompressor    compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native compression=none                  -rate threads=20 -pop seq=1..50000000    -log interval=5"]
```

Comment:
Same for these.
Pick one/two.

Reply:
I'll remove all of them, and leave one read, to start with